lib/sync/installation.js

```diff
@@ -86,9 +86,17 @@ module.exports.processInstallation = (app, queues) => {
         app.log(message);
 
         try {
-          await jiraClient.devinfo.migration.complete();
+          /**
+           * Only call github/migrationComplete in prod. Complete will only be called in Jira if
+           * GITHUB_CONNECT_APP_IDENTIFIER is equal to com.github.integration.production
+           */
+          if (process.env.NODE_ENV === 'production') {
+            await jiraClient.devinfo.migration.complete();
+          } else {
+            app.log('Successfully completed installing GitHub org.');
```

Comment:
this will only get logged if we're in non-prod?

Reply:
It should be any environment other than prod right?

Reply:
well, I guess it depends?  Right now, in prod, you'll have the migration actually complete, but no log, while in anything but prod, you get the log but it's a 'lie' because there was no migration.  I guess I'm questioning if it's needed at all for non-prod environments or should it always display in any environment?